src/com/android/mms/ui/SlideListItemView.java

```diff
@@ -257,9 +257,12 @@ public void setVideo(String name, Uri video) {
             mAttachmentName.setText("");
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit